packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -146,20 +145,8 @@ This may indicate that the patch contains native changes, which cannot be applie
 
     try {
       final shouldCodesign = argResults['codesign'] == true;
-      final (flutterVersionAndRevision, flutterVersion) = await (
-        shorebirdFlutter.getVersionAndRevision(),
-        shorebirdFlutter.getVersion(),
-      ).wait;
-
-      if ((flutterVersion ?? minimumSupportedIosFlutterVersion) <
-          minimumSupportedIosFlutterVersion) {
-        logger.err(
-          '''
-iOS patches are not supported with Flutter versions older than $minimumSupportedIosFlutterVersion.
-For more information see: ${supportedFlutterVersionsUrl.toLink()}''',
-        );
-        throw ProcessExit(ExitCode.software.code);
-      }
```

Comment:
Seems super restrictive to only allow patching and releasing using the latest stable version on iOS (when we introduced this customers were pretty frustrated) but happy to revert this 🤷

Reply:
I personally think it's ok for folks to patch older releases especially since they can stage and preview the patches 🤷 
It causes more harm than good to be so restrictive imo since we disincentive engagement/usage.